jvm/src/test/scala/com/raquo/domtypes/GeneratorSpec.scala

```diff
@@ -90,6 +90,31 @@ class GeneratorSpec extends AnyFunSpec with Matchers {
     )
   }
 
+  it("Generate MathMl tags") {
+    println("=== MATHML TAGS ===")
+
+    val traitName = "MathMlTags"
+
+    val fileContent = generator.generateTagsTrait(
+      tagType = MathMlTagType,
+      defGroups = defGroups.mathMlTagsDefGroups,
+      printDefGroupComments = false,
+      traitCommentLines = Nil,
+      traitModifiers = Nil,
+      traitName = traitName,
+      keyKind = "MathTag",
+      baseImplDefComments = Nil,
+      keyImplName = "mathTag",
```

Comment:
`mathTag` name is already taken for the <math> tag, this should be something else. I think`keyImplName` should be `mathMlTag`.